bftengine/src/bftengine/ClientsManager.cpp

```diff
@@ -268,6 +268,7 @@ void ClientsManager::markRequestAsCommitted(NodeIdType clientId, ReqId reqSeqNum
   if (reqIt != requestsInfo.end()) {
     reqIt->second.committed = true;
     LOG_DEBUG(GL, "Marked committed" << KVLOG(clientId, reqSeqNum));
+    return;
```

Comment:
Hi @shgandhi , I don't understand how this change helps us prevent the error message from the log?
`2021-02-28T12:15:42,334Z|ERROR|6|concord.bft|message-processing||250378|ClientsManager.cpp:273|void bftEngine::impl::ClientsManager::markRequestAsCommitted(bftEngine::impl::NodeIdType, bftEngine::impl::ReqId)|Request not found clientId: 17, reqSeqNum: 6771764798733418496|{"primary":"0","path":"fast"}|`